docs/api.md

```diff
@@ -164,6 +164,15 @@ operation.  One of:
   * *power management error* -- The host could not be powered on or
     off, as requested through the *online* field setting.
 
+*softPowerOffStatus* -- The status of the soft power off command.
+Value is one of the following:
+  * *<empty string>* -- Soft power off is not used on the node.
+  * *unsupported* -- Soft power off is not supported on the node.
+  * *triggered* -- Soft power off is triggered on the node but
```

Comment:
Do we really need to track the soft power off status separately?

If we do, I think instead of reflecting it in a new status field, we should see if we can combine it with the provisioning status and make that a top level field on the status structure.

We would never have something in "provisioning" with a soft power off status of "triggered", for example, right?